src/coreclr/tools/aot/crossgen2/CommandLineOptions.cs

```diff
@@ -239,6 +239,10 @@ public CommandLineOptions(string[] args)
                     extraHelp.Add(archString.ToString());
                 }
 
+                extraHelp.Add("");
+                extraHelp.Add(SR.CpuFamilies);
+                extraHelp.Add(string.Join(", ", Internal.JitInterface.InstructionSetFlags.AllCpuFamilies()));
```

Comment:
Could you please apply the same fix for ilc (NativeAOT compiler)? https://github.com/dotnet/runtime/blob/10288c966bfdd46cdc0d612beaa05888819171a9/src/coreclr/tools/aot/ILCompiler/Program.cs#L306 ?